packages/@okta/courage-for-signin-widget.js

```diff
@@ -0,0 +1,11372 @@
+/*! DO NOT MODIFY THIS FILE MANUALLY. IT IS GENERATED FROM PACKAGE @OKTA/COURAGE-FOR-SIGNIN-WIDGET. */
```

Comment:
Not a blocker, but we should consider removing comments and uglifying this file. We'd reduce this file down to about `140000` bytes, an improvement by nearly 63%.